index.html

```diff
@@ -13,7 +13,9 @@
 
       <div class="main-body">
         <nav id="navbar">
-          <header>Front-End Technologies Web Development Documentation</header>
+          <header><h2>
```

Comment:
- Nice wok, I suggest you use h2-h6 tag in the header

Reply:
Review: Please use heading tags h2 - h6 in all your header elements

Reason: This practice is good because it will help you to better structure your work. Use descriptive headings and titles because they help a user navigate their browser and the page.